simpleLooping/createListPowerOfBase.py

```diff
@@ -11,3 +11,21 @@
 # Ex:
 #   createListPowerOfBase(2, 4) # returns [1, 2, 4, 8]
 #   createListPowerOfBase(3, 3) # returns [1, 3, 9]
+from typing import List
+ 
+def createListPowerOfBase(base: int, powerCap: int) -> List[int]:
+    if powerCap == 0:
+        return []
+
+    if powerCap == 1:
```

Comment:
This is over engineer and you don't need this if. When you run the code without the if, it will give you the same result, give a try.